model/project_ref.go

```diff
@@ -106,8 +106,7 @@ type ProjectRef struct {
 	PerfEnabled        *bool                        `bson:"perf_enabled,omitempty" json:"perf_enabled,omitempty" yaml:"perf_enabled,omitempty"`
 
 	// This is a temporary flag to enable individual projects to use repo settings
```

Comment:
This comment should change now (or be removed entirely) since it's referencing UseRepoSettings.